compiler/src/dmd/id.d

```diff
@@ -553,7 +553,7 @@ immutable Msgtable[] msgtable =
     { "_align", "align" },
     { "aligned" },
     { "__pragma", "pragma" },
-    { "builtins", "__builtins" },
+    { "builtins", "__importc_builtins" },
```

Comment:
From what I understand of the comment the problem is the other way around.

```
{ "importc_builtins", "__builtins" },
```

Reply:
@JohanEngelen what do you say?

Reply:
given `{ "foo", "bar" }`, `foo` is the name used  in the DMD source code (`Id.foo`), `bar` is the name in D source code (`module bar;`)

Reply:
I actually meant both:
- `builtins` as "module" name in DMD source code is very confusing for something that only relates to importC. I  like Iain's suggestion for a fix.
- the file name `__builtins.di` is unfortunate for importC, so I really like that this PR fixes that.

Reply:
```suggestion
    { "importc_builtins", "__importc_builtins" },
```

Reply:
ok